Packages/vcs/Lib/Canvas.py

```diff
@@ -4718,10 +4718,7 @@ def ffmpeg(self, movie, files, bitrate=1024, rate=None, options=''):
             cmd += '-i %s_%%d.png' % (rnd)
         elif isinstance(files, str):
             cmd += '-i ' + files
-        if rate is not None:
-            cmd += ' -r %s ' % rate
-        if bitrate is not None:
-            cmd += ' -b:v %sk' % bitrate
+        cmd += " -pix_fmt yuv420p "
```

Comment:
Yup, FFMPEG defaults to using h264 when libx264 is installed. That's the reason for the -pix_fmt yuv420p option (they default to a pixel format that quicktime doesn't support but most other players do, so I went with the lowest common denominator).